src/components/AppSidebar/AppSidebar.vue

```diff
@@ -804,38 +814,6 @@ $top-buttons-spacing: 6px;
 					font-size: var(--default-font-size);
 				}
 			}
-
-			// favourite
-			.app-sidebar-header__star {
-				display: block;
-				width: $clickable-area;
-				height: $clickable-area;
-
-				.icon-loading-small {
-					display: block;
-					width: $clickable-area;
-					height: $clickable-area;
-				}
-
-				.star {
-					width: $clickable-area;
-					height: $clickable-area;
-					&--star {
-						color: #000;
-						opacity: .5;
-						&:hover {
-							color: #FC0;
-						}
-					}
-					&--starred {
-						color: #FC0;
-						&:hover {
-							color: #000;
-							opacity: .5;
-						}
```

Comment:
Is the hover behavior moved somewhere else?

Reply:
It's now using a Button, so maybe not needed anymore?

Reply:
Yes not needed as it should behave as Button component